assets/src/components/vehicleIcon.tsx

```diff
@@ -42,6 +49,34 @@ export interface TooltipProps {
   vehicleOrGhost: VehicleOrGhost
 }
 
+export const vehicleOrientation = (
+  vehicle: VehicleOrGhost,
+  ladderDirections: LadderDirections
+): Orientation => {
+  if (vehicle.routeId !== null && vehicle.directionId !== null) {
+    const ladderDirection: LadderDirection = getLadderDirectionForRoute(
+      ladderDirections,
+      vehicle.routeId
+    )
+    const vehicleDirection: VehicleDirection = directionOnLadder(
+      vehicle.directionId,
+      ladderDirection
+    )
+
+    if (vehicle.routeStatus === "laying_over") {
```

Comment:
It seems to! To test I changed this line to`(vehicle.routeStatus === "a")` and got the error `This condition will always return 'false' since the types 'RouteStatus' and '"a"' have no overlap.`